CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ fixing this.
 
 # HEAD (changes that will go into the next release)
 
+# v3.0.0
+
+Bug fixes:
+* Use OS-specific libs when computing client User-Agent in kubectl, etc. (https://github.com/kubernetes/kubernetes/pull/44423)
+* kubectl commands run inside a pod using a kubeconfig file now use the namespace specified in the kubeconfig file, instead of using the pod namespace. If no kubeconfig file is used, or the kubeconfig does not specify a namespace, the pod namespace is still used as a fallback. (https://github.com/kubernetes/kubernetes/pull/44570)
+* Restored the ability of kubectl running inside a pod to consume resource files specifying a different namespace than the one the pod is running in. (https://github.com/kubernetes/kubernetes/pull/44862)
+* Fix a bug that caused invalid Tolerations to be applied to Pods created by DaemonSets. (https://github.com/kubernetes/kubernetes/pull/45349)
```

Comment:
Removed. The changes affect client-go/pkg/api/v1/helpers.go, but don't really affect the correctness of client-go.